sdk/state/close.go

```diff
@@ -0,0 +1,120 @@
+package state
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/stellar/experimental-payment-channels/sdk/txbuild"
+	"github.com/stellar/go/txnbuild"
+	"github.com/stellar/go/xdr"
+)
+
+// The steps for a channel coordinated close are as followed:
+// 1. Initiator or Responder submits latest declaration tx
+// (in steps 2-4 Initiator and Responder are interchangeable, as long as they alternate)
+// 2. Initiator calls ProposeCoordinatedClose
+// 3. Responder calls ConfirmCoordinatedClose
+// 4. Initiator calls ConfirmCoordinatedClose
+
+type CoordinatedClose struct {
+	observationPeriodTime      time.Duration
+	observationPeriodLedgerGap int64
```

Comment:
These periods of time should be zero for a coordinated close. Why are they required?

Reply:
will a coordinated close always have a zero observation time? I wasn't sure if a shorter O (not necessarily zero) should be allowed

looking at the sep again though that it what is says. I can change

Reply:
https://github.com/stellar/experimental-payment-channels/pull/95/commits/e00b88157d4faf1510792a74ce340487cdec34fc